tests/queries/0_stateless/02668_ulid_decoding.sql

```diff
@@ -0,0 +1,3 @@
+-- Tags: no-fasttest
+
+SELECT dateDiff('s', ULIDStringToDateTime(generateULID()), now()) = 0;
```

Comment:
Let's also add tests for invalid arguments, wrong arguments types and invalid ULID (with different sizes)